README.md

```diff
@@ -164,4 +221,4 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+This would be a level 1 Chomsky. This is because certain non-terminals on either side of terminals are not required but the terminals are required, making this a context-sensitive grammar.
```

Comment:
But you've already shown you can implement it using a state machine, which makes it type 3...

# -3